browser_tests/tests/menu.spec.ts

```diff
@@ -178,6 +178,75 @@ test.describe('Menu', () => {
       await comfyPage.menu.topbar.triggerTopbarCommand(['ext', 'foo-command'])
       expect(await comfyPage.getVisibleToastCount()).toBe(1)
     })
+
+    test('Can navigate Theme menu and switch between Dark and Light themes', async ({
```

Comment:
How do you feel about throwing in some `.screenshot()`, not necessarily as assertions, but for local runs at least.